angular-app/src/app/components/home/home.component.html

```diff
@@ -1,87 +1,157 @@
+            </div>
+            <div fxLayout="column" fxFlexAlign="space-evenly">
+              <div fxLayout="row"
+                   fxFlexAlign="center"
+                   class="metric-information">{{metrics.openChannels}}
+              </div>
+              <div fxLayout="row"
+                   fxFlexAlign="center"
+                   class="metric-description">Open Channels
+              </div>
+            </div>
+          </div>
         </div>
 
-        <div class="item-column">
-          <img src="../../../assets/users_icon.png">
+        <div fxLayout="column">
+          <div fxLayout="row" fxLayoutGap="0.8em">
+            <div fxLayout="column" fxLayoutAlign="center center" fxFlex="4em">
+              <img src="../../../assets/users_icon.png">
+            </div>
+            <div fxLayout="column" fxFlexAlign="space-evenly">
+              <div fxLayout="row"
+                   fxFlexAlign="center"
+                   class="metric-information">{{metrics.uniqueUsers}}
+              </div>
+              <div fxLayout="row"
+                   fxFlexAlign="center"
+                   class="metric-description">Unique Users
+              </div>
+            </div>
+          </div>
         </div>
 
-        <div class="item-column">
-          <div class="item-row metric-information">{{metrics.uniqueUsers}}</div>
-          <div class="item-row metric-description">Unique Users</div>
-        </div>
       </div>
 
-      <div class="item-row">
-        <div class="item-column">
-          <a href="http://raiden-network.readthedocs.io/en/stable/webui_tutorial.html#tokens" class="link-button">
+      <div class="information-buttons"
+           fxLayout="row"
+           fxLayout.lt-md="column"
+           fxLayoutAlign="center"
+           fxFlexAlign.lt-md="center"
+           fxLayoutGap="0.3rem">
+        <div fxLayout="column"
+             fxLayout.lt-md="row"
+             fxFlex="0 0 200px"
+             fxFlex.lt-md="0 0 auto">
+          <a mat-stroked-button
+             color="primary"
+             [ngStyle.lt-md]="{'width': '100%'}"
+             href="http://raiden-network.readthedocs.io/en/stable/webui_tutorial.html#tokens"
+             matTooltip="Learn how to register a token">
             Token registration tutorial
-            <span class="tooltiptext">Learn how to register a token</span></a>
+          </a>
         </div>
-        <div class="item-row">
-          <a href="http://raiden-network.readthedocs.io/en/stable/webui_tutorial.html#channels" class="link-button">
+        <div fxLayout="column"
+             fxLayout.lt-md="row"
+             fxFlex="0 0 200px"
+             fxFlex.lt-md="0 0 auto">
+          <a mat-stroked-button
+             color="primary"
+             [ngStyle.lt-md]="{'width': '380px'}"
+             matTooltip="Learn how to open a channel"
+             href="http://raiden-network.readthedocs.io/en/stable/webui_tutorial.html#channels">
```

Comment:
Loredana asked to make these links open in a new tab, can you add this: https://stackoverflow.com/questions/15551779/open-link-in-new-tab-or-window

Reply:
I was planning on making them on the next PR :)